examples/pytorch_lightning_simple.py

```diff
@@ -139,13 +144,13 @@ def objective(trial):
     # PyTorch Lightning will try to restore model parameters from previous trials if checkpoint
     # filenames match. Therefore, the filenames for each trial must be made unique.
     checkpoint_callback = pl.callbacks.ModelCheckpoint(
-        os.path.join(MODEL_DIR, 'trial_{}'.format(trial.number)))
+        os.path.join(MODEL_DIR, 'trial_{}'.format(trial.number)), save_best_only=False)
```

Comment:
I added `save_best_only=False` because the default value has been changed since `pytorch-lightning==0.5.3`. 
I found that `ModelCheckpoint` did not save any models when `save_best_only` was set to `True`.

0.5.2.1  (save_best_only=False)
https://github.com/williamFalcon/pytorch-lightning/blob/0.5.2.1/pytorch_lightning/callbacks/pt_callbacks.py#L178

0.5.3 (save_best_only=True)
https://github.com/williamFalcon/pytorch-lightning/blob/0.5.3/pytorch_lightning/callbacks/pt_callbacks.py#L179

Reply:
Thank you for the investigation. The changes LGTM, I'm just verifying your fix locally.

Reply:
It worked locally.